src/systems/filecoin_blockchain/storage_power_consensus/_index.md

```diff
@@ -5,6 +5,8 @@ entries:
 - storage_power_actor
 ---
 
+TODO: remove all stale .id, .go files referenced
+
 {{<label storage_power_consensus>}}
 The Storage Power Consensus subsystem is the main interface which enables Filecoin nodes to agree on the state of the system. SPC accounts for individual storage miners' effective power over consensus in given chains in its {{<sref power_table>}}. It also runs {{<sref expected_consensus>}} (the underlying consensus algorithm in use by Filecoin), enabling storage miners to run leader election and generate new blocks updating the state of the Filecoin system.
```

Comment:
Total network power states should live here https://github.com/filecoin-project/specs-actors/blob/eac38c1d153cb985fa147c97113b28718361c90b/actors/builtin/power/power_state.go#L17

Reply:
Thanks @zixuanzh. cc @mishmosh , this is probably a deeper question related to how the spec should acknowledge or integrate specs-actors

Reply:
SPC does not account for individual power